src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutesSpec.scala

```diff
@@ -68,7 +68,7 @@ class ManagedGroupRoutesSpec extends AnyFlatSpec with Matchers with ScalatestRou
 
   // Makes an anonymous object for a user acting on the same data as the user specified in samRoutes
   def makeOtherUser(samRoutes: TestSamRoutes, samUser: SamUser = defaultNewUser) = new {
-    runAndWait(samRoutes.userService.createUser(samUser, samRequestContext))
+    samRoutes.createUserAndAcceptTos(samUser, samRequestContext)
```

Comment:
`samRoutes.createUserAndAcceptTos` creates the user and accepts the ToS. These are two different calls, and both are called with `runAndWait`. The two actions were pulled into one method because we need to create user that has accepted the ToS all over our tests.